pkg/core/fs_ro_ops.go

```diff
@@ -233,28 +260,47 @@ func (txns *populateFSTxns) commitToFS(fs *readOnlyFsInternal) {
 	fs.fsDirStore = txns.dirStore.Commit()
 }
 
-/* unwound recursion to build a list of ents terminating at the first extant parent */
+func newFSTxns(fs *readOnlyFsInternal) *populateFSTxns {
+	return &populateFSTxns{
+		dirStore:     fs.fsDirStore.Txn(),
+		lookupTree:   fs.lookupTree.Txn(),
+		fsEntryStore: fs.fsEntryStore.Txn(),
+	}
+}
+
+type populate struct {
```

Comment:
`populateFSAddBundleEntryParams`?

Reply:
No I specifically don't want to adopt the `myLargerThanLifeTypeNaming` convention.

Similarly, I renamed many functions with long names.
You are welcome to suggest a more appropriate name, it only has to be short and meaningful 

Basically, this is the same trend as for package naming.
You might find this useful: https://blog.golang.org/package-names
https://rakyll.org/style-packages/

Reply:
follow-up note: disagree - won't fix